Class/Druid/Spells/Moonfire.cpp

```diff
@@ -12,6 +12,8 @@
 #include "SimSettings.h"
 #include "Utils/Check.h"
 
+#include <cmath>
```

Comment:
Can you please move this up between the implemented class header and its CSIM dependencies? As per [the code style in the wiki](https://github.com/timhul/ClassicSim/wiki/Code-Style#header-inclusion-order-and-forward-declarations). So this would be:

```c
#include "Moonfire.h"

#include <cmath>

#include "Buff.h"
...
```

(the same for the other inclusions in this change)